src/applications/static-pages/facilities/vetCentersHours.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// TODO fix spacing and add unit tests
+const VetCenterHours = props => {
+  const arrayOfWeekdays = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+
+  const formatTime = hoursString => {
+    // starthours: 700, endhours: 1730
+    let hour = Number(
+      hoursString.toString().length > 3
+        ? hoursString.toString().substring(0, 2)
+        : hoursString.toString().substring(0, 1),
+    );
+    const minutes = Number(
+      hoursString.toString().length > 3
+        ? hoursString.toString().substring(2, 4)
+        : hoursString.toString().substring(1, 4),
+    );
+    let AMorPM = 'a.m.';
+    if (hour >= 0 && hour <= 24 && minutes >= 0 && minutes <= 60) {
+      if (hour > 12) AMorPM = 'p.m.';
```

Comment:
If we return `null` after failing the sanity check on line 21, I think we can use a const instead of a let here:

```suggestion
      const AMorPM = hour > 12 ? 'a.m.' : 'p.m.';
```